src/style.css

```diff
@@ -7,12 +7,14 @@ main {
   font-family: "Poppins", sans-serif;
```

Comment:
- [ ] You have done an excellent job with your To-Do list structure :muscle: but I would suggest updating a few things about the styling to make your To-Do list look even better :fire:
According to the design, the only solid borders are the ones inside (or between the rows) because the To-Do container has a shadow :wink:
  Template:
  ![image](https://user-images.githubusercontent.com/81522435/145122993-be22fb7b-1217-4c2d-b8bc-bc6489e7850f.png)

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/145123043-70ca6713-f249-4346-9124-8ac49a030ae2.png)

- [ ] Also, the input field should not have a solid border and it should take the whole width until the icon on the right side. You could try with `border: none` for the `<input>` :smiley:
  ![image](https://user-images.githubusercontent.com/81522435/145123359-2ccb8a51-a306-4256-8daa-1b5a02a43e97.png)
  ![image](https://user-images.githubusercontent.com/81522435/145123382-3c823a3e-c8d3-49b3-88c1-e1c671f54aa9.png)